lib/saluki-event/src/eventd/mod.rs

```diff
@@ -1,9 +1,33 @@
-//! Event types
-mod priority;
-pub use self::priority::*;
+//! Events.
 
-mod alert;
-pub use self::alert::*;
+// TODO: Switch usages of `String` to `MetaString` since we should generally be able to intern these strings as they
+// originate in in the DogStatsD codec, where interning is already taking place.
```

Comment:
Usually we'll try to keep TODO comments close to the area where they're most relevant, but in this case, this is a thing we'd want to apply to multiple fields on the `EventD` struct, so we just shove it up here as a general TODO.